Guess game/Program.cs

```diff
@@ -80,10 +93,8 @@ static int StartGame()
 
         private static void GiveAnswer(bool correct)
```

Comment:
improve this method so it would handle all 3 cases (Teisinga, Dar kart, Spejimai isnaudoti) + move color changing in this method 
(hint: you will need 2 parameters to this  method: one for deciding how to differentiante between dar kart and spejimai isnaudoti & another one to pass which color should be set for which text) 
after this method update in you code it should be only 1 line of `Console.ForegroundColor =` & `Console.ResetColor();`